bind/plan.sh

```diff
@@ -27,18 +30,30 @@ pkg_bin_dirs=(
 pkg_include_dirs=(include)
 pkg_lib_dirs=(lib)
 
+ply_version="3.11"
+
 do_prepare() {
   # The configure script expects `file` binaries to be in `/usr/bin`
   if [[ ! -r /usr/bin/file ]]; then
     ln -sv "$(pkg_path_for file)/bin/file" /usr/bin/file
     _clean_file=true
   fi
+
+  pip install \
+    --target "${pkg_prefix}/pip" \
+    --ignore-installed \
+    "ply==${ply_version}"
 }
 
 do_build() {
-  ./configure --prefix="${pkg_prefix}" \
+  attach
```

Comment:
Looks like an attach got left in :) Should be good to go after that is gone.

Reply:
Goddamn it! 😆